packages/tf-inventory/tests/stateFile.js

```diff
@@ -177,6 +183,19 @@ test('listTfDeployments lists unique Tf deployments based on state file name', (
   t.deepEqual(deployments, ['cumulus', 'tf']);
 });
 
+test('listTfDeployments lists unique Tf deployments based on state file name with specified regex', (t) => {
+  const stateFiles = [
+    'bucket/cumulus/terraform.tfstate',
+    'bucket/cumulus/terraform.tfstate',
```

Comment:
Why repeat the same value in the state files? Would this be a better test if this second entry was `bucket2/cumulus/terraform.tfstate`? Based on your regex, I think `cumulus` would still be returned as the deployment name. This is actually mildly confusing in reality - because state files in different buckets may have the same "deployment name" but clearly are artifacts from different deployments. However, it would prove that `listTfDeployments` is respecting the specified regex, which is what the test is trying to prove.

Do we already have another test proving the case where all state files are listed as their own deployments? That would seem like a good counterpart to this test with the same scenario of state files for the same deployment name across multiple buckets

Reply:
That was a copy/paste error. Deployments with the same name across different buckets would be listed as one deployment. Historically this has been a good thing because it generally means someone changed the state file location and one needs to get deleted. It shows up in the deployment report.

Reply:
This was a copy/paste error btw. It should not have had those dups in there. thanks for catching that.